src/std/text/json/output.ss

```diff
@@ -63,6 +64,8 @@
                (write-json-vector obj output env))
               ((hash-table? obj)
                (write-json-hash obj output env))
+              ((walist? obj)
+               (write-json-alist (walist->alist obj) output env))
```

Comment:
do we care about deduplication of keys or is it guaranteed to not have duplicates?

Reply:
If the user uses the lower level alist, I'd say he's on his own, but if you want we can check for duplicates indeed (and triple check the keys are strings or such indeed)

Reply:
No, this is not the right place to check for duplicates -- it should be in the walist level operations.
And agreed, if the user mucks with the alist directly, he is asking for trouble.